### PR TITLE
Capistrano setup

### PR DESCRIPTION
**Pull request by bsonntag**
Setups Capistrano to deploy Botler to Vagrant.

### CODE REVIEW

**Comment by bsonntag** [comment]
@naps62 Please review.

File: config/deploy/staging.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# server 'example.com',
+#   user: 'user_name',
+#   roles: %w{web app},
+#   ssh_options: {
+#     user: 'user_name', # overrides user setting above
+#     keys: %w(/home/user_name/.ssh/id_rsa),
+#     forward_agent: false,
+#     auth_methods: %w(publickey password)
+#     # password: 'please use keys'
+#   }
```
Comment (naps62) [inline config/deploy/staging.rb]: shouldn't this file (and the one above) be deleted?

Comment (bsonntag) [inline config/deploy/staging.rb]: Sure. Done.

**Comment by bsonntag** [comment]
Closing this because the project has been inactive for a long time.

**Pull request closed without merging**